Update README files for new directory layout

tjohns opened the pull request.

tjohns committed:
Update README files for new directory layout

silvolu commented on samples/android/gcm-demo/README.md:
```diff
@@ -0,0 +1,62 @@
+
+## Screenshots
+
+![Menu](samples/android/gcm-demo/screenshots/menu.png "The app's menu")
```
The screenshot are still broken:
https://github.com/tjohns/gcm/tree/master/samples/android/gcm-demo

tjohns replied:
Done.

silvolu commented on README.md:
```diff
@@ -1,45 +1,18 @@
-This project contains an Android app that demonstrates these aspects of GCM:
+This project contains client libraries and samples to help developers interface
+with and explore the Google Cloud Messaging APIs.
 
```
Do you want to add a reference to the quickstarts here as well?

tjohns replied:
Done.

tjohns committed:
Fix paths for gcm-demo README screenshot

Now that the screenshots are localized to this file, their
paths shouldn't be relative to the root of the repo.

tjohns committed:
Add quickstart links to /README.md
